docs/_quarto.yml

```diff
@@ -325,6 +314,23 @@ quartodoc:
             - DecimalValue
             - name: BooleanValue
               package: ibis.expr.types.logical
+            - name: ifelse
```

Comment:
I think I would put `ifelse` in generic.

It's closer to the notion of "branching", part of which is of course boolean logic related.

Perhaps a good rule of thumb is:

- If the operation returns or operates on a large variety of types it should go in generic
- Otherwise it should go in the specific docs

I am aware that `ifelse` is defined on bool expressions :)

Reply:
Sure! I would vote for putting it in both sections, instead of moving it. What do you think?

Reply:
I think I'd rather have it in single place so that there aren't multiple search results that could be confusing. I see this with sphinx sometimes and I really dislike it.

Reply:
OK, now it is just in generic in my latest commit stack.